elyra/pipeline/processor_local.py

```diff
@@ -115,6 +115,13 @@ def operation_name(self) -> str:
     def process(self, operation: Operation):
         raise NotImplementedError
 
+    @staticmethod
+    def _collect_envs(operation: Operation) -> Dict:
+        envs = os.environ.copy()  # Make sure this process's env is "available" in the kernel subprocess
+        envs['ELYRA_RUNTIME_ENV'] = "local"  # Special case
```

Comment:
Wouldn't it be safer to reference the `_type` class variable https://github.com/elyra-ai/elyra/blob/f17143b3561e0226b3c53de2fcd4a398fa34ff0a/elyra/pipeline/processor_local.py#L42 instead of hard-coding the value?

Reply:
`_type` is not on an `OperationProcessor` - which is where "local" runtimes perform env collection.  I could pass this as a parameter, but it (at least today) will never be used outside of a local "runtime".  Had we treated "local" as a full-fledged runtime (eh um), then we'd be able to use "type" - as I do in `RuntimePipelineProcess._collect_envs()`.